core/user/user.go

```diff
@@ -54,10 +54,10 @@ func NewUUID() (UUID, error) {
 // Validate returns an error if the UUID is invalid.
 func (u UUID) Validate() error {
 	if u == "" {
-		return fmt.Errorf("empty uuid")
+		return fmt.Errorf("%wempty uuid", errors.Hide(errors.NotValid))
```

Comment:
Elsewhere we've been putting the %w to hide at the end of the string. Can we agree on which way we want t do it and be consistent. I prefer at the end as it doesn't interfere with the user reading the string as they peruse the code.

Reply:
Done & Done